CONTRIBUTING.md

```diff
@@ -124,9 +128,16 @@ If you've found something in the content or the website that should be updated,
 
 ### Labels
 
-Labels can help you find a perfect issue for your skills.
+We use labels to keep track on how the PR is going, some examples:
+
+waiting for review - PR is waiting for developer review
+works for me: QA team has tested your PR
+changes-requested: Someone from our team identified that something needs to be changed/fixed on your PR
+stale: PR which haven't seen any changes for last 3 days and its prone to closing
+blocked: PR is currently blocked and cannot be merged
+
```

Comment:
visual-ok missing

Reply:
also would be nice to divide the "labels" into issue labels and pr labels 


mention p1,p2, p3, $ signs and maybe area tags

Reply:
> visual-ok missing

added

Reply:
> also would be nice to divide the "labels" into issue labels and pr labels
> 
> mention p1,p2, p3, $ signs and maybe area tags

changed the text section because there was a p1-p5 section already